content/ai/concepts/ai-ethics/ai-ethics.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'AI Ethics'
+Description: 'The study of how AI systems ought to be developed, distributed, and used.'
+Subjects:
+  - 'AI'
+Tags:
+  - 'AI'
+  - 'Accessibility'
+  - 'Deep Learning'
+  - 'Neural Networks'
+  - 'Justice'
+CatalogContent:
+  - 'learn-explainable-ai-xai'
+  - 'intro-to-ai-strategy'
+---
+
+**AI ethics** is the study of ethical issues that arise in the development, distribution, and use of AI systems. As AI-based technologies become more powerful and accurate, they will have a large influence on many aspects of human societies. Already, AI systems are being deployed to help make decisions in finance, law, transportation, employment, and many other areas. It is therefore extremely important for those involved in the development and distribution of AI technologies to carefully consider ethical issues that will arise in the use of these technologies.
+
+Below is a very brief overview of many major ethical issues arising from the growth of AI. The overview is loosely ordered based on immediacy: it begins with issues that already exist and ends with issues are only beginning to arise or may arise in the future. Issues are listed with a definition when necessary, and then a set of central questions arising within that issue. Each of these issues is discussed in greater detail in the term pages below.
+
+## Major Ethical Issues in AI
+
+### Algorithmic Bias 
```

Comment:
@noahpgordon, the verify formatting check is failing because there is an extra space in the text. To resolve this, please remove the extra space.

```suggestion
### Algorithmic Bias